Foundation/NSURLSession/HTTPMessage.swift

```diff
@@ -141,12 +141,17 @@ extension URLSessionTask {
 
 extension URLSessionTask._HTTPMessage {
     var headersAsDictionary: [String: String] {
-        var result: [String: String] = [:]
-        headers.forEach {
-            result[$0.name] = $0.value
-        }
-        return result
-    }
+		var result: [String: String] = [:]
```

Comment:
Hi @NobodyNada , it'd be great if you could change the identation as @xwu pointed out. Thanks.